firebase/firebase.json

```diff
@@ -0,0 +1,35 @@
+{
+  "firestore": {
+    "rules": "firestore.rules",
+    "indexes": "firestore.indexes.json",
+    "predeploy": [
+      "npm --prefix \"$PROJECT_DIR\" run firestore-rules-test"
+    ]
+  },
+  "functions": {
+    "source": "src/functions",
+    "predeploy": [
+      "npm --prefix \"$PROJECT_DIR\" run functions-build"
+    ]
+  },
+  "emulators": {
+    "storage": {
+      "port": 9199
+    },
```

Comment:
Firebase Storage emulator was launched recently and still have a lot of bugs. I would remove it while it's not stable